src/main/java/seedu/address/storage/JsonAdaptedPolicyList.java

```diff
@@ -0,0 +1,62 @@
+package seedu.address.storage;
```

Comment:
Perhaps we should rename this file/class to JsonSerializablePolicyList?

Reply:
Ok sounds good too. Can change it.